storage/backend/ceph/ceph_test.go

```diff
@@ -182,6 +184,26 @@ func (s *cephSuite) TestMounted(c *C) {
 			},
 		},
 	})
+
+	output, err := exec.Command("sudo", "rbd", "lock", "--format", "json", "list", driverOpts.Volume.Name, "--pool", "rbd").Output()
+	c.Assert(err, IsNil)
```

Comment:
I think logging the command output in case of assert failures can be use. We can use `Commentf()` as the last argument to `Assert()` for the same.